commands/adminsonline.js

```diff
@@ -22,18 +43,32 @@ module.exports = {
 
             rcon.on("connect", () => console.log("connected"));
             rcon.on("authenticated", () => console.log("authenticated"));
+            rcon.on("error", () => console.log("errors"));
             rcon.on("end", () => console.log("end"));
 
-            await rcon.connect();
-
-            let responses1 = await rcon.send(rconToSend);
+            try {	// tests to see if the connection works, if not fails to catch block 
 
-            msg.channel.send(responses1);
+                await rcon.connect();
+                const responses = await rcon.send(rconToSend);
+                rcon.end();
```

Comment:
shood be 
```suggestion
                await rcon.end();
``` as its a promise

Reply:
oof github markup failed me

Reply:
I dont see a way to edit this to accept it without the ``` as its a promise part.

Reply:
```suggestion
                rcon.end();
                await rcon.end();
```

Reply:
lol now you have rcon.end() and the rcon.end again

Reply:
```suggestion
                await rcon.end();
```